var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+from spack.util.executable import which
```

Comment:
This is not needed.

Reply:
That's a good point - it works without the import, but I don't understand why actually `which(...)` works, I'd expect something like `util.executable.which(...)`

Reply:
It's part of Todd's Spack "magic".  The pkgkit.py file contains all of the imports that are automatically made available to packages.  Technically `from spack import *` isn't even needed.